docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -67,7 +71,9 @@ You should now be able to see Redis data flowing in on index `metrics-redis.info
 
 NOTE: All assets (dashboards, ingest pipelines, and so on) related to the Redis integration are not installed. You need to explicitly <<install-uninstall-integration-assets,install them through {kib}>>.
 
-To set the target host dynamically for a targeted Pod based on its labels, use a variable in the {agent} policy to return path information from the provider:
+Conditions can also be used in the `Kubernetes` package in order to set the target host dynamically for a targeted Pod based on its labels.
```

Comment:
by `Kubernetes package` are you referring to the https://docs.elastic.co/en/integrations/kubernetes ?

for standalone we usually use in documentation `input` instead of package, for example - [here](https://observability-docs_2552.docs-preview.app.elstc.co/guide/en/fleet/master/elastic-agent-input-configuration.html) or [here](https://observability-docs_2552.docs-preview.app.elstc.co/guide/en/fleet/master/conditions-based-autodiscover.html#conditions-based-autodiscover)

can we keep it consistent?

Reply:
thank you for clarifications, I don't have a strong opinion on it

Reply:
No strong opinion either :)

Reply:
Ok as long you don't have a strong opinion I decided to use `input` instead of package to keep consistency because I am not sure if the user at this point is 100% familiar with packages.